app/cdap/components/ConnectionList/Components/HeaderSearchInputField/index.tsx

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import { IHeaderSearchInputFieldProps } from 'components/ConnectionList/types';
+import React, { ChangeEvent } from 'react';
+
+export default function({ type, classnames, refs, onChange, index }: IHeaderSearchInputFieldProps) {
```

Comment:
plz use styled variations of input field and move this input field component to components/common inside the feature file

Reply:
Created a styled component in common folder as discussed